.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# migrating to black
+a61e51b53612735e93b3bb8a7605030c499cd6c7
```

Comment:
Maybe also add the flynt commit? 

Commit id: 7a335e3ef297057b4a12f7c21d4e8dd426b44ca0
#3786

Reply:
yeah, sure, let's do that!